include/entity.h

```diff
@@ -0,0 +1,122 @@
+#ifndef ENTITY_H
+#define ENTITY_H
+
+#include "common.h"
+
+typedef s32 EntityScript[0];
+typedef s32 EntityModelScript[0];
+
+enum {
+    ENTITY_SCRIPT_OP_End,
+    ENTITY_SCRIPT_OP_Jump,
+    ENTITY_SCRIPT_OP_Call,
+    ENTITY_SCRIPT_OP_SetCallback,
+    ENTITY_SCRIPT_OP_Goto,
+    ENTITY_SCRIPT_OP_Label,
+    ENTITY_SCRIPT_OP_RestartBoundScript,
+    ENTITY_SCRIPT_OP_SetFlags,
+    ENTITY_SCRIPT_OP_ClearFlags,
+    ENTITY_SCRIPT_OP_PlaySound,
+};
+
+enum {
+    ENTITY_MODEL_SCRIPT_OP_End,
+    ENTITY_MODEL_SCRIPT_OP_Draw,
+    ENTITY_MODEL_SCRIPT_OP_Restart,
+    ENTITY_MODEL_SCRIPT_OP_Loop,
+    ENTITY_MODEL_SCRIPT_OP_SetRenderMode,
+    ENTITY_MODEL_SCRIPT_OP_SetFlags,
+    ENTITY_MODEL_SCRIPT_OP_ClearFlags,
+    ENTITY_MODEL_SCRIPT_OP_DrawImage,
+};
+
+#define es_End ENTITY_SCRIPT_OP_End,
+#define es_Jump(script) ENTITY_SCRIPT_OP_Jump, (s32)script,
+#define es_Call(func) ENTITY_SCRIPT_OP_Call, (s32)func,
+#define es_SetCallback(func, time) ENTITY_SCRIPT_OP_SetCallback, time, (s32)func,
+#define es_Goto(labelId) ENTITY_SCRIPT_OP_Goto, labelId,
+#define es_Restart ENTITY_SCRIPT_OP_Goto, 0,
```

Comment:
Unsure what I think of having a pseudoinstruction like this. Evt scripts have a similar case where `evt_Loop(0)` is an infinite loop, but we decided against adding an `evt_LoopForever`-like macro there. It does create nice parity with `ems_Restart` however

Reply:
`es_Goto(0)` without corresponding `es_Label(0)` looks a bit confusing